en/frames_rover/aion_r1.md

```diff
@@ -0,0 +1,78 @@
+# Aion Robotics R1 UGV
+
+<Badge type="warning" text="main (PX4 v1.15)" />
+
+This vehicle was chosen to test and improve the differential drive support for PX4.
+
+Currently, PX4 supports MANUAL mode for this setup.
```

Comment:
Worth making this obvious up front.

Reply:
How would you recommend doing that? Should I make it more Bold, or add it in the rease notes?

Reply:
Sorry, I mean "I moved your comment up to the introduction to make it more obvious" (i.e. I was just explaining why I moved your comment about manual mode).